backend/routes/post.py

```
@@ -144,6 +144,36 @@ def send_post_notifications(db: Session, user: User, post: Post):
 
     db.commit()  # ✅ Commit once after all notifications are added
 
+@router.get("/{post_id}")
+def get_single_post(
```

Ruff (0.8.2)

Missing return type annotation for public function `get_single_post`

(ANN201)